packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -100,10 +100,15 @@ export class MessageWidget extends Component {
 
   componentWillUpdate(nextProps) {
     const props = this.props;
+
     if (this.activityList
       && nextProps.conversation.get(`activities`).count() !== props.conversation.get(`activities`).count()) {
       this.scrollHeight = this.activityList.getScrollHeight();
     }
+
+    if (!nextProps.features.hasIn([`items`, `js-widgets-mentions`])) {
+      props.getFeature(`developer`, `js-widgets-mentions`, nextProps.sparkInstance);
```

Comment:
`getFeature` isn't imported or in proptypes

Reply:
Oh wow, guess `getFeature` was getting passed by the container. I'll add that into this widget so we don't run into odd issues.